ansible/roles/probe-services/templates/probe-services-nginx

```diff
@@ -0,0 +1,104 @@
+# ansible-managed in ooni-sysadmin.git
+
+{% import 'common.j2' as c %}
+
+server {
+    listen 80;
+    listen 443 ssl;
+
+    {{ c.ssl_letsencrypt(probe_services_domain) }}
+
+    server_name {{ probe_service_domain }};
+    access_log  /var/log/nginx/{{ probe_services_domain }}.access.log;
+    error_log   /var/log/nginx/{{ probe_services_domain }}.error.log warn;
+
+
+    # The below paths have been obtained by looking at the access.log for the
+    # hosts and running:
+    # cat access.log | cut -d ' ' -f8 | sort | uniq
+
+    # Bouncer
+    # Should match:
+    # - /bouncer
+    # - /bouncer/net-tests
+    location /bouncer {
+        proxy_http_version 1.1;
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_read_timeout 900;
+
+        proxy_pass http://{{ bouncer_ipv4 }}:{{ bouncer_port }};
+    }
+
+    # Collector
+    # Should match:
+    # - /report
+    # - /report/XXXX
+    # - /report/XXXX/close
+    # - /invalidpath
+    location ~^/(report|invalidpath) {
+        proxy_http_version 1.1;
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_read_timeout 900;
+
+        proxy_pass http://{{ collector_ipv4 }}:{{ collector_port }};
+    }
+
+    # Web Connectivity Test Helper
+    # Should match:
+    # - /
+    # - /status
+    #
+    # The fact that it responds to / means that we may have to differentiate
+    # via the Host record.
+    # TODO We should check if clients will respect a suffix added to by the
+    # bouncer in the returned field, otherwise new clients should use another
+    # form
+    location ~^/web-connectivity/(status) {
```

Comment:
@bassosimone do you know the answer to the above question for measurement-kit?

Basically if I return to a measurement-kit probe instead of:
```
      - {address: 'https://b.web-connectivity.th.ooni.io:443', type: https}
```

I say:

```
      - {address: 'https://b.web-connectivity.th.ooni.io:443/web-connectivity', type: https}
```

Will the client logic know to append the suffix web-connectivity or not?

Reply:
Unfortunately it seems like this strategy will not work for legacy OONI Probe clients (see: https://github.com/ooni/probe-legacy/blob/master/ooni/backend_client.py#L52), so we will most likely still have to do `Host` header based routing.

Reply:
Maybe we can see to improve the behaviour for new clients?

Reply:
> Maybe we can see to improve the behaviour for new clients?

Yeah for sure. We probably want new clients to also be aware of the fact that they can speak any of the probe services APIs to any of the hosts (i.e. that the backends are oonified).

Reply:
https://github.com/ooni/probe-engine/issues/63